tests/tasks/create_bond_profile.yml

```diff
@@ -0,0 +1,49 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Include network role
+  include_role:
+    name: linux-system-roles.network
+  vars:
+    network_connections:
+      # Create a bond controller
+      - name: "{{ controller_profile }}"
+        state: up
+        type: bond
+        interface_name: "{{ controller_device }}"
+        bond:
+          mode: 802.3ad
+          ad_actor_sys_prio: 65535
+          ad_actor_system: 00:00:5e:00:53:5d
+          ad_select: stable
+          ad_user_port_key: 1023
+          all_ports_active: true
+          downdelay: 0
+          lacp_rate: slow
+          lp_interval: 128
+          miimon: 110
+          min_links: 0
+          num_grat_arp: 64
+          primary_reselect: better
+          resend_igmp: 225
+          updelay: 0
+          use_carrier: true
+          xmit_hash_policy: encap2+3
```

Comment:
Not sure if it was clear, these options also belong in a variable in the playbook.

Reply:
Aha, thanks for making it clearer, I will use a variable to hold the bond options. Bonus from this, I think that we can have fewer task files.